marklogic-data-hub-central/ui/e2e/setup.sh

```diff
@@ -1,16 +1,25 @@
 #!/bin/bash
-set e+x
+
+cd qa-project;
+DHS=`echo $1 | cut -d'=' -f 2`
+mlHost=`echo $2 | cut -d'=' -f 2`
+env='local'
 
 credentials='-PmlUsername=dh-dev -PmlPassword=dh-dev'
-cd qa-project
 
 ./gradlew hubInit
-cp ../cypress/fixtures/users/* src/main/ml-config/security/users/
-./gradlew mlDeploy --info
-./gradlew hubSaveIndexes --info
-./gradlew hubDeployAsDeveloper $credentials --info
 
+if $DHS
+then
+        ./gradlew hubDeployAsDeveloper  --info $credentials -PenvironmentName=dhs -PmlHost=$mlHost
+        env='dhs'
+else
+        cp ../cypress/fixtures/users/* src/main/ml-config/security/users/
+        ./gradlew mlDeploy --info
+        ./gradlew hubSaveIndexes --info
+        ./gradlew hubDeployAsDeveloper $credentials --info
+fi
 
-./gradlew hubRunFlow $credentials -PflowName=AdvantageFlow -PentityName=Customer -PbatchSize=100 -PthreadCount=4 -Psteps="1,2"
-./gradlew hubRunFlow $credentials -PflowName=PersonFlow -PentityName=Person -PbatchSize=100 -PthreadCount=4 -Psteps="1,2"
-./gradlew hubRunFlow $credentials -PflowName=PersonXMLFlow -PentityName=PersonXML -PbatchSize=100 -PthreadCount=4 -Psteps="1,2"
+./gradlew hubRunFlow $credentials -PenvironmentName=$env -PflowName=AdvantageFlow -PentityName=Customer -PbatchSize=100 -PthreadCount=4 -Psteps='1,2' -PmlHost=$mlHost  --info
```

Comment:
Are we going to provide mlHost param even when running locally?
ie `./setup.sh dhs=false mlHost=localhost`

Reply:
Yes